Release 4.0.0

rlepinski opened the pull request:
Updates to SDK 16.0.3+

Ulrico972 commented on Pod/Classes/AirshipGimbalAdapter.swift:
```diff
@@ -101,43 +101,44 @@ import Gimbal
-        let regionEvent: UARegionEvent = UARegionEvent(regionID: visit.place.identifier, source: source, boundaryEvent: .enter)!
-        UAirship.shared().analytics.add(regionEvent)
+        if let regionEvent = RegionEvent(regionID: visit.place.identifier,
+                                      source: source,
```
kind of weird space here

Ulrico972 reviewed (approved):
just a weird space but it's good anyway

rlepinski committed:
Release 4.0.0

# Conflicts:
#	AirshipGimbalAdapter.podspec
#	CHANGELOG
#	Pod/Classes/AirshipGimbalAdapter.swift

rlepinski committed:
Update version, fix changelog

# Conflicts:
#	AirshipGimbalAdapter.podspec
#	CHANGELOG
